Automattic/wp-calypso
# Update WP Courses banners to WP Learn

#### Proposed Changes

* Updates copy and assets of WP Courses task and card to reflect the switch to WP Learn.

Before | After
--- | ---
<img width="1510" alt="Screenshot 2023-01-26 at 2 41 33 PM" src="https://user-images.githubusercontent.com/1398304/214967191-30d47421-22f9-4560-9f76-e7c900298e86.png"> | <img width="1510" alt="Screenshot 2023-01-26 at 2 16 45 PM" src="https://user-images.githubusercontent.com/1398304/214966954-3fbe2dc6-5d91-444d-9442-ae2f95bed267.png">
<img width="1466" alt="Screenshot 2023-01-26 at 2 41 48 PM" src="https://user-images.githubusercontent.com/1398304/214967211-28503eff-0003-4705-9cfa-a220b057c71d.png"> | <img width="1510" alt="Screenshot 2023-01-26 at 2 36 27 PM" src="https://user-images.githubusercontent.com/1398304/214967233-7d559185-eea5-4713-9cb7-4cfaf68fa21d.png">


#### Testing Instructions



* Checkout this PR or use the Calypso.live link in the comments.
* Navigate to a new site's Home screen.
* Verify the links work as expected, the logos look good etc.

#### Pre-merge Checklist






Fixes #72682
